sycl/include/CL/sycl/handler.hpp

```diff
@@ -426,6 +426,13 @@ class handler {
     KernelFunc(detail::Builder::getItem<Dims, true>());
   }
 
+  template <typename KernelName, typename KernelType, int Dims,
+            EnableIfNDItem<KernelType, Dims> = 0>
```

Comment:
Do you really need enable if here?

Reply:
> Do you really need enable if here?

I think We need EnableIfNDItem here since we have to make sure that parallel_for(nd_range)'s kerneltype is nd_item.